supervisor/api/ingress.py

```diff
@@ -279,7 +279,7 @@ async def _handle_request(
             try:
                 response.headers["X-Accel-Buffering"] = "no"
                 await response.prepare(request)
-                async for data in result.content.iter_chunked(4096):
+                async for data in result.content.iter_any():
```

Comment:
```suggestion
                async for data, _ in client.content.iter_chunks():
```

Reply:
I thought iter_any was the fastest though?